tests/api_connexion/endpoints/test_connection_endpoint.py

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import unittest
+
+import pytest
+
+from airflow.www import app
+
+
+class TestConnectionEndpoint(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+        cls.app = app.create_app(testing=True)  # type:ignore
+
+    def setUp(self) -> None:
+        self.client = self.app.test_client()  # type:ignore
```

Comment:
This class code is repeated in each test file. Also we have `http_client` fixture which also provides the app 😸

Reply:
Yes. There are so many repetitions in this code because it is a skeleton. I hope that soon this code will be more different. I didn't want to introduce level 3 inheritance here, because it would complicate the tests excessively. This is ok if the code is repeated because it makes the code much simpler and less complex.
Here is an article about it: https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html

Reply:
Here is a real example: https://github.com/apache/airflow/blob/69c546fb0fa779f2f311e5a96c98900ef997afce/tests/api_connexion/endpoints/test_dag_model_endpoint.py

Reply:
And what about the fixture? Should it stay or not?

Reply:
I removed fixture: https://github.com/apache/airflow/pull/8149/commits/6c765df9afc067574e29bfc262c0ec6e10427dce
Thanks.